### DISCUSSION TITLE
Not found page in infinite loop when deployed to Vercel

### DISCUSSION THREAD

**Discussion by dspedroborges** [vercel/community#7351] [Help]
### Summary

The not found page of my app (treinomix.vercel.app) gets in an infinite loop of refreshing, even when I'm not using a custom not found page. This happens only on the vercel server, when I deploy the project. At localhost it works just fine. What could it be?

### Steps to Reproduce

Every non existing pages causes this infinite loop of refreshing.

**Comment by pawlean** [comment]
Hi @dspedroborges!

Are you referring to this page? I just typed a random string after `treinomix.vercel.app/` 

![CleanShot 2024-07-03 at 13 25 11@2x](https://github.com/vercel/community/assets/17087373/69bb518f-06e4-48cc-8863-d675777198cb)

Unsure how to investigate without looking at your code, but have you tried creating a custom 404 page to see if that prevents this?

**Reply by dspedroborges** [reply]
Yes, I have tried to create a custom 404 page and it doesn't prevent this. It only happens when it's on Vercel server. Running on localhost it works just fine.

[a comment by an automated account is not shown]
